src/configurations/destinations/appcues/ui-config.json

```diff
@@ -162,6 +162,10 @@
                   "name": "Custom",
                   "value": "custom"
                 },
+                {
+                  "name": "iubenda",
```

Comment:
Wouldn't this be `Iubenda`? The official name.

Reply:
Thank you for bringing this up, but we always write in lowercase: “iubenda”